src/components/QuickStart/index.tsx

```diff
@@ -0,0 +1,214 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import './index.scss';
+
+import { Card } from '@rmwc/card';
+import { GridCell } from '@rmwc/grid';
+import { CollapsibleList, SimpleListItem } from '@rmwc/list';
+import { Typography } from '@rmwc/typography';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { createStructuredSelector } from '../../store';
+import { Config } from '../../store/config';
+import { getConfig } from '../../store/config/selectors';
+import { squash } from '../../store/utils';
+import { Spinner } from '../common/Spinner';
+
+export const mapStateToProps = createStructuredSelector({
+  configRemote: getConfig,
+});
+
+export type PropsFromState = ReturnType<typeof mapStateToProps>;
+export type Props = PropsFromState;
+
+export const Home: React.FC<Props> = ({ configRemote }) =>
+  squash(configRemote, {
+    onNone: () => <Spinner span={12} message="Loading QuickStart Guide..." />,
+    onData: config => <QuickStart config={config} />,
+    onError: () => <QuickStart config={{}} />,
+  });
+
+export default connect(mapStateToProps)(Home);
+
+const QuickStart: React.FC<{
+  config: Partial<Config>;
+}> = ({ config }) => {
+  return (
+    <>
```

Comment:
Open question: Zero state: What do we show when this list contains nothing? For example, when only `hosting` and `pubsub` is started.

Idea: full screen card with a link to official Emulator Suite docs?

Reply:
Yeah that sounds good